pkg/agent/openflow/client.go

```diff
@@ -403,12 +403,13 @@ func (c *client) InstallNodeFlows(hostname string,
 	localGatewayMAC := c.nodeConfig.GatewayConfig.MAC
 
 	for peerPodCIDR, peerGatewayIP := range peerConfigs {
-		if peerGatewayIP.To4() != nil {
+		isIPv6 := peerGatewayIP.To4() != nil
```

Comment:
ditto, peerConfigs should be changed to two paramters and the loop should be removed to avoid confusion.

Reply:
Updated.